src/uwtools/apps/srw.py

```diff
@@ -40,8 +40,8 @@ def load_config(self, config_file: str) -> None:
                     ">%s" % f,
                 ]
                 cmd = " ".join(cmd_components)
-                result = execute(cmd=cmd)
-                if not result.success:
+                _, success = execute(cmd=cmd)
```

Comment:
Previously, `execute()` returned a `NameSpace` object with `.output` and `.success` attributes, but these did not carry any useful type information. That functions has been updated to return a `Tuple[bool, str]`-typed value that _does_ carry useful type information, and is now unpacked into two local variables wherever it is called.